tests/kubernetes_test.py

```diff
@@ -37,7 +37,10 @@ def mock_kubernetes_task():
 
 @pytest.fixture
 def mock_kubernetes_cluster():
-    with mock.patch("tron.kubernetes.PyDeferredQueue", autospec=True,), mock.patch(
+    with mock.patch(
+        "tron.kubernetes.PyDeferredQueue",
+        autospec=True,
+    ), mock.patch(
```

Comment:
Surprised this isn't already formatted like this. How old of black are we running?

Reply:
Looks like black 24.8.0 (last version of black that supports py38) also formats it this way.